docs/doxygen-user/multi-user/installSolr.dox

```diff
@@ -0,0 +1,141 @@
+/*! \page install_solr_page Install and Configure Solr
+
+\section install_solr_overview Overview
+
+Autopsy uses Apache Solr to store keyword text indexes.  A central server is needed in a multi-user cluster to maintain and search the indexes.  
+
+A new Solr core is created for each case and the core/index is stored in the case folder (not on the local drive of the Solr server). 
```

Comment:
I would propose changing this to "A new text index is created for each case and is stored in the case folder on shared storage (not on the local drive of the Solr server).